### PR TITLE
Provide recipe for python2, using 2.3.x releases of secretstorage

### PR DESCRIPTION
**Pull request by yarikoptic**
Needed for DataLad 0.10.0 package python2 build

I guess this one is not really to be merged into master but be taken as 2.x branch to live within the repository...

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
This should target a branch, not master

**Comment by yarikoptic** [comment]
I know, but I saw no way to change to any other (new) branch somehow in the PR interface.  How "to target a branch" then?

**Comment by yarikoptic** [comment]
and hold on - I screwed up I guess -- it did build for 3.5 and 3.6 instead of 2.7... looking into it now

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: True  # [win or osx or py<35]
+  skip: True  # [win or osx or py>32]
```
Comment (yarikoptic) [inline recipe/meta.yaml:14]: was there anything else to do to make it build for 27 instead of all the 3*?

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by yarikoptic** [comment]
eh, crap... there seems to be no other dbus (pydbus?) python bindings in conda-forge, that is why I guess there were no python2 package and only the 3.x one which uses jeepney instead :-(  not yet sure what to do

**Comment by mariusvniekerk** [comment]
Odds are we require the upstream python library.  If you want to submit it to staged recipes we can work that there, and resume this when that lands

**Comment by yarikoptic** [comment]
I guess it would need to be this one  http://www.freedesktop.org/wiki/Software/DBusBindings#Python  - not sure if I would have enough juice for it now...  most likely I would need to just come up with some nasty workaround (well -- just skip some tests ;-)), at the cost of lost functionality, in DataLad for now

**Comment by yarikoptic** [comment]
I guess I will close it for now since there is no immediate resolution unfortunately.  Thanks everyone who helped!

**Comment by mariusvniekerk** [comment]
it shouldn't be that hard since we have the `dbus` recipe around

**Comment by ccordoba12** [comment]
> eh, crap... there seems to be no other dbus (pydbus?) python bindings in conda-forge

There are Python bindings for dbus in conda-forge already (else this package couldn't work at all). They are provided by the Jeepney package. Since Jeepney is Python 3 only, that's why this package is for Python 3 too.

The great advantage of Jeepney over previous bindings is that is pure Python. There has been a couple of efforts before to compile `python-dbus` (which works for Python 2), but they have failed (I think it requires a lot of Gtk dependencies). @yarikoptic, you can find those in staged-recipes.

**Comment by yarikoptic** [comment]
Thank you @ccordoba12 ! I had a pick at Debian package - didn't require any thing but dbus, but had some custom settings passed... I might look around tomorrow while traveling

**Comment by jakirkham** [comment]
So `secretstorage` itself [seems to indicate it is Python 3 only]( https://github.com/mitya57/secretstorage/blob/master/setup.py#L35 ). Also it is the one [requiring `jeepney`]( https://github.com/mitya57/secretstorage/blob/master/setup.py#L36-L37 ) and this feedstock is merely mirroring that requirement. Given this, not sure it makes much sense to attempt a Python 2 build until some changes are made upstream.

**Comment by yarikoptic** [comment]
2.x series of it was python 2 compatible and is still recommended for it

**Comment by jakirkham** [comment]
Gotcha, thanks for clarifying.

**Pull request closed without merging**